debian/changelog

```diff
@@ -3,6 +3,8 @@ cloud-init (23.4.1-0ubuntu1~20.04.3) UNRELEASED; urgency=medium
   * d/apport-general-hook.py: Move apport hook to main branch
   * d/cloud-init.maintscript: remove /etc/cloud/clean.d/README
   * d/cloud-init.logrotate: add logrotate config for cloud-init
+  * d/cloud-init.templates: enable WSL datasource by default
+  * d/p/keep-dhclient-as-priority-client.patch
```

Comment:
Was this one missed previously?

Reply:
Yeah I missed it in https://github.com/canonical/cloud-init/pull/4923/files#diff-93467241211d3f9d253fb8ad341738fbda5e5d3ade4721326f2bba82525fbaee